### PR TITLE
TSK-1490: Fix autocomplete in workplace workbasket search

### PR DESCRIPTION
**Pull request by sofie29**
https://sonarcloud.io/dashboard?branch=TSK-1490&id=sofie29_taskana

---
 
### For the submitter:
- [ ] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [x] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [x] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [ ] I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I put the ticket in review
- [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken

### Verified by the reviewer:
- [ ] Commit message format → TSK-XXX: Your commit message.
- [ ] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [ ] SonarCloud analysis meets our standards
- [ ] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [ ] PR fulfills the ticket
- [ ] Edge cases and unwanted side effects are tested
- [ ] Readability

### CODE REVIEW

File: web/src/app/workplace/components/task-list-toolbar/task-list-toolbar.component.ts [hunk 1]
```diff
@@ -77,6 +78,13 @@ export class TaskListToolbarComponent implements OnInit {
     }
   }
 
+  filterWorkbasketNames() {
```
Comment (Franzi321) [inline web/src/app/workplace/components/task-list-toolbar/task-list-toolbar.component.ts:81]: The filter function doesn't work if you type in numbers.. It seems to work if you're using the following function. 

  filterWorkbasketNames() {
    this.filteredWorkbasketNames = this.workbasketNames.filter((value) =>
    value.toLowerCase().includes(this.resultName.toLowerCase())
    );
  }

Comment (sofie29) [reply web/src/app/workplace/components/task-list-toolbar/task-list-toolbar.component.ts:81]: It worked with numbers but the user had to fill in the name from its beginning, e.g. 'basxet0' wasn't found when 'xet0' was typed in. Your solution solves this problem, thank you!

Comment (Franzi321) [reply web/src/app/workplace/components/task-list-toolbar/task-list-toolbar.component.ts:81]: :) 
The filterValue variable seems to be unused now..

Comment (sofie29) [reply web/src/app/workplace/components/task-list-toolbar/task-list-toolbar.component.ts:81]: yes 😄

**Review by Franzi321 — APPROVED** [review]

**Pull request merged**